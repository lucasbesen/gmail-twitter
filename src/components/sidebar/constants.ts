export const SIDEBAR_ITEM_TYPE = {
  INBOX: 'INBOX',
  STARRED: 'STARRED',
  SNOOZED: 'SNOOZED',
};
