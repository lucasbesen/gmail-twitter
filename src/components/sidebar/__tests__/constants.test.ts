import { SIDEBAR_ITEM_TYPE } from '../constants';

it('should return SIDEBAR_ITEM_TYPE constant', () => {
  expect(SIDEBAR_ITEM_TYPE).toMatchSnapshot();
});
