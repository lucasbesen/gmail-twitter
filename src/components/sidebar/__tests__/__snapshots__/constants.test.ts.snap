// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return SIDEBAR_ITEM_TYPE constant 1`] = `
Object {
  "INBOX": "INBOX",
  "SNOOZED": "SNOOZED",
  "STARRED": "STARRED",
}
`;
